src/console_cmds.cpp

```diff
@@ -1049,6 +1049,54 @@ DEF_CONSOLE_CMD(ConNewGame)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConNewHeightMapGame)
+{
+	bool heightmap_loaded = true;
+
+	if (argc == 0) {
+		IConsoleHelp("Load a game by name or index. Usage: 'newheightmapgame [<file | number> [seed]]'");
+		IConsoleHelp("The server can force a new game using 'newheightmapgame'; any client joined will rejoin after the server is done generating the new game.");
+	}
+
+	if (argc >=1 && argc <= 3) {
```

Comment:
Missing spacing. This could be changed to an early exit if argc < 1 || argc > 3, reducing the indentation depth below.

Reply:
That leads to your question regarding returning `true` on `argc == 0`, and if that is good or bad. Returning `false` for `argc < 1` would be inconsistent with existing commands of major importance (to my eyes) like `ConNewGame`.

Reply:
`argc == 0` has a very specific meaning, which is "the user requested help on the command". It should always return true, and print the help message, for that case.

Reply:
Well that means the 'early exit' style would not trigger on `argc <1 || argc > 3`, but on `argc > 3`, like what I already did.
The question about using an 'early exit' style or merely setting the return variable value for a single `return` instruction shall be part of discussion around line 1095 below.